include/OpenColorIO/OpenColorIO.h

```diff
@@ -161,6 +161,17 @@ extern OCIOEXPORT void ResetToDefaultLoggingFunction();
 /// Log a message using the library logging function.
 extern OCIOEXPORT void LogMessage(LoggingLevel level, const char * message);
 
+/**
+ * @brief Set the Compute Hash Function object to use; otherwise, use the
```

Comment:
[Minor] For now, the header is using `\brief` instead of `@brief`.

Reply:
Fixed.